src/test/controller_mapping_validation_test.cpp

```diff
@@ -93,10 +93,10 @@ bool lintMappingInfo(const MappingInfo& mapping) {
         result = false;
     }
 
-    if (mapping.getWikiLink().trimmed().isEmpty()) {
-        qWarning() << "LINT:" << mapping.getPath() << "has no wiki link.";
-    } else if (!checkUrl(mapping.getWikiLink())) {
-        qWarning() << "LINT:" << mapping.getPath() << "has invalid wiki link";
+    if (mapping.getManualLink().trimmed().isEmpty()) {
+        qWarning() << "LINT:" << mapping.getPath() << "has no manual link.";
+    } else if (!checkUrl(mapping.getManualLink())) {
+        qWarning() << "LINT:" << mapping.getPath() << "has invalid manual link";
```

Comment:
checking a couple mapping XMLs, the tag in question doesn't actually contain a link, just a name for a page. Am I missing something?